reflection/app/Models/Company.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\QueryException;
+
+class Company extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'Name',
+        'email',
+        'logo',
+        'website',
+
+    ];
+
+    protected function Employee()
```

Comment:
relation method should be camelCase

Reply:
Also, as this is a has many, I'd suggest making the relation name `employees` (plural) as this just makes it that bit clearer whether you can expect a model or a collection back when using it